ui/src/components/TaskDetails.tsx

```diff
@@ -28,13 +32,32 @@ const TaskDetails: React.FC<{}> = () => (
           return <ErrorCallout error={error} />
         case RequestStatus.READY:
           if (data) {
+            const hasAra =
+              get(data, "adaptiveResourceAllocation", false) === true
             return (
               <>
                 <ViewHeader
                   breadcrumbs={[
                     { text: "Tasks", href: "/tasks" },
                     {
-                      text: data.alias || definitionID,
+                      text: (
+                        <div style={{ display: "flex" }}>
+                          <div>{data.alias || definitionID}</div>
+                          {hasAra && (
+                            <Tooltip content="blah blah blah cool new feature">
```

Comment:
@ujjwalsfix

Reply:
Adaptive CPU and Memory resource allocation based on prior run history